functions/go/starlark/krmfn/krmfn.go

```diff
@@ -0,0 +1,31 @@
+package krmfn
+
+import (
+	"github.com/GoogleContainerTools/kpt-functions-sdk/go/fn"
+	"go.starlark.net/starlark"
+	"go.starlark.net/starlarkstruct"
+)
+
+const ModuleName = "krmfn.star"
+
+var Module = &starlarkstruct.Module{
+	Name: "krmfn",
+	Members: starlark.StringDict{
+		"match_gvkn": starlark.NewBuiltin("match_gvkn", matchGVKN),
+	},
+}
+
+func matchGVKN(thread *starlark.Thread, _ *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	var resourceList starlark.Value
+	var apiVersion, kind, name string
+	if err := starlark.UnpackPositionalArgs("match_gvkn", args, kwargs, 4,
```

Comment:
We can support match GVK or GVKN or GVKNN. We choose to match GVKN here while our golang SDK choose to match GVK.
@yuwenma What's your opinions? IIRC you implemented matching GVK in golang SDK.

Reply:
GVK is type and NN is meta. the GVKNN  or GVKN is a short form mainly used in kustomize, my main question is how easy it is for users to understand the N or NN. @natasha41575 Do you know how popular this GVKNN convention among kustomize users? If not, separate match_gvk and match_namespace might be easier to use.

Reply:
I'll change this to have separate match_gvk, match_name, and match_namespace so that it is easier to use.

Reply:
done!